app/authoring/integrating-yeoman.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: documentation
+title: Integrating Yeoman
+category: authoring
+sidebar: sidebars/authoring.html
+excerpt: How can you integrate Yeoman system in your own tools
+---
+
+Everytime you run a generator, you're actually using the `yeoman-environment`. This environment is a base system who's decoupled from any UI component and can be abstracted away by any tool. When you run `yo`, you're basically just running a terminal UI façade on top of the core yeoman environment.
+
+## The basics
+
+What you need to know first is that the environment system is contained in the `yeoman-environment` package. You can install it:
```

Comment:
`The first thing you need to know is the environment system is provided by the `yeoman-environment` package.`